### DISCUSSION TITLE
Can we use html content to show a html page in email

### DISCUSSION THREAD

**Discussion by c-a-narain** [resend/react-email#1656] [Q&A]
Is there any component or ways to display Html content in the email? I got the contents with the html tags in it as in the reference image. 

![image](https://github.com/user-attachments/assets/44cc069a-3add-4c92-af35-f0f33582a07f)

**Comment by gabrielmfern** [comment]
You can use [dangerouslySetInnerHtml](https://legacy.reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml)
